### PR TITLE
[TH2-4262] Added external_user_queue field to batches metadata

### PR DESCRIPTION
**Pull request by Nikita-Smirnov-Exactpro**

### CODE REVIEW

**Comment by OptimumOpium** [comment]
I think we should use `dev` branch as a base branch

File: src/main/proto/th2_grpc_common/common.proto [hunk 1]
```diff
@@ -42,6 +42,14 @@ message MessageID {
   repeated uint32 subsequence = 4; // List of nested sequences in addition to the main one
 }
 
+message EventBatchMetadata {
+  string external_user_queue = 1; // External user queue can be used for routing by internal box logic.
```
Comment (ivandruzhinin) [inline src/main/proto/th2_grpc_common/common.proto:46]: Maybe rename `response_queue`?

Comment (Nikita-Smirnov-Exactpro) [reply src/main/proto/th2_grpc_common/common.proto:46]: I'd save external. Because this queue isn't under infra control

**Comment by Nikita-Smirnov-Exactpro** [comment]
Changes has been moved into https://github.com/th2-net/th2-grpc-common/pull/48 PR

**Pull request closed without merging**